schema/deploy/tables/budget_item.sql

```diff
@@ -0,0 +1,46 @@
+-- Deploy cif:tables/budget_item to pg
+
+begin;
+create table cif.budget_item
+(
+  id integer primary key generated always as identity,
+  amount numeric(20,2),
```

Comment:
This might be something we can confirm with the business area. For this first version we can probably leave it unrestricted